utils/__init__.py

```diff
@@ -0,0 +1,185 @@
+import numpy as np
```

Comment:
I am not sure where to put ``grp_converter``, `AndersonAcceleration`, and `spectral_norm`.
Any suggestion?